requirements/requirements.txt

```diff
@@ -18,7 +18,7 @@ django-kombu==0.9.1
 django-rosetta==0.7.1
 django-tastypie==0.9.14
 djtables==0.1.1
-egenix-mx-base==3.2.8
+#egenix-mx-base==3.2.8
```

Comment:
@dmyung do you recall what we needed egenix for? Relevant commits below:

ce69b72 [2 years, 4 months ago] re-add egenix-mx-base [Mike White]
9c3eb57 [2 years, 4 months ago] add note about egenix-mx-base to readme [Mike White]
7bf49a9 [2 years, 4 months ago] revert egenix-mx-base requirements.txt addition [Mike White]
2297948 [2 years, 4 months ago] updating requirements - explicit psycopg req, plus egenix-base [dmyung]
57288e4 [2 years, 4 months ago] add egenix-mx-base to requirements.txt [Mike White]